app/controllers/application.js

```diff
@@ -152,6 +152,8 @@ export default Controller.extend(cancelOrderMixin, quantityUpdateMixin, {
     },
 
     hideCart() {
+      this.updateRequestedQuantityValue(this.get("updatedQuantity"));
```

Comment:
name `updatedQuantity` says it is a number but the method `updateRequestedQuantityValue` is expecting an object. It's very ambiguous.